### PR TITLE
Fixed bug where removeTempArea would be called on home even if it was not defined

### PR DESCRIPTION
**Pull request by MitchelPaulin**
Ran into an issue where after poping a graph from a dock area I would get an error about calling removeTempArea on None. This seems to have fixed the problem.

### CODE REVIEW

**Comment by j9ac9k** [comment]
I really should sync up develop branch w/ master, ... as the current state of develop is rather broken.  I should also update the contributing guide to suggest creating PRs to merge with master.  Anyway let me fix the target branch and let CI rerun and do its thing...

**Comment by MitchelPaulin** [comment]
@j9ac9k Everything look good?

**Comment by j9ac9k** [comment]
Sorry I'm a bit off the grid as I'm actually moving.  I probably won't get to look at this for at least another week.  Paging @ixjlyons or @2xB  to take a look if they have time.

**Comment by j9ac9k** [comment]
Hi @MitchelPaulin sorry it's taken me so long to follow up.  Moving takes forever... now that I'm taking a closer look at this, I have one suggestion, which I'll make in-line.

Thanks again for the PR and sorry for the delayed reply.

File: pyqtgraph/dockarea/DockArea.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def apoptose(self, propagate=True):
         #print "apoptose area:", self.temporary, self.topContainer, self.topContainer.count()
         if self.topContainer is None or self.topContainer.count() == 0:
             self.topContainer = None
-            if self.temporary:
+            if self.temporary and self.home:
```
Comment (j9ac9k) [inline pyqtgraph/dockarea/DockArea.py:322]: would suggest having `and self.home is not None` if we're checking for `None` and not the boolean condition of `self.home`

**Comment by j9ac9k** [comment]
LGTM!  Thanks for the change! Merging!

**Pull request merged**